Makefile

```diff
@@ -213,6 +213,7 @@ shell: build-dirs build-env
 		-v "$$(pwd)/.go/std/$(GOOS)/$(GOARCH):/usr/local/go/pkg/$(GOOS)_$(GOARCH)_static:delegated" \
 		-v "$$(pwd)/.go/go-build:/.cache/go-build:delegated" \
 		-v "$$(pwd)/.go/golangci-lint:/.cache/golangci-lint:delegated" \
+		-v "$$(pwd)/.gitconfig:/root/.gitconfig:ro" \
```

Comment:
For the build container, the `WORKDIR` is set to `/root`, but it'd be better to use something like `$HOME` instead of `/root`.